pkg/sql/colexec/COLEXEC.bzl

```diff
@@ -1,44 +1,50 @@
-def _generate_impl(ctx):
-    output_files, input_files = [], []
-    # Gather the full list of all input template files (deconstructing it from
-    # the filegroup input). See [1] for a reference of all types/members within
-    # skylark.
+def generate(name, template, output, misc = []):
```

Comment:
wonder if we should give this a better name, i.e. `generate_colexec`

Reply:
Done.